tools/update-postject.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# Shell script to update postject in the source tree to the latest release.
+
+# This script must be in the tools directory when it runs because it uses the
+# script source file path to determine directories to work in.
+
+set -ex
+
+cd "$( dirname "$0" )/.." || exit
```

Comment:
```suggestion
ROOT="$( dirname "$0" )/.."
cd "$ROOT" || exit
```

Reply:
@aduh95 this is mostly a copy paste of `tools/update-undici.js`, so I think it would be a better idea to do this change in both the scripts in one shot in a separate PR. Also, I have moved the Postject addition change to its own PR - https://github.com/nodejs/node/pull/45066, so if you have further comments about that, feel free to post it there!